### PR TITLE
feat: add get help menu item

### LINKED ISSUE #2823: Add app-wide option for getting support
Users currently have no way to find support from within the wallet itself. We should add an app-wide way to find it, even if later we add more specific contextual prompts based on particular flows or errors.

For example, we could add it in the app menu or header:

image

It could link to our support FAQ, where we intend to add more content (e.g. where to submit a support ticket): https://wallet.hiro.so/wallet-faq/where-can-i-find-support-for-the-stacks-wallet

cc @314159265359879

### PR DESCRIPTION
**Pull request by edu-stx**
> Try out this version of the Hiro Wallet - download [extension builds](https://github.com/hirosystems/stacks-wallet-web/actions/runs/3540130614).

Closes https://github.com/hirosystems/stacks-wallet-web/issues/2823

image

### CODE REVIEW

File: src/app/features/settings-dropdown/settings-dropdown.tsx [hunk 1]
```diff
@@ -101,6 +102,21 @@ export function SettingsDropdown() {
...
               Change theme
             </MenuItem>
+            <MenuItem
+              data-testid={SettingsSelectors.ToggleTheme}
+              onClick={wrappedCloseCallback(() => {
+                void analytics.track('get_support_menu_item');
+                window.open(
+                  'https://wallet.hiro.so/wallet-faq/where-can-i-find-support-for-the-stacks-wallet',
+                  '_blank'
+                );
```
Comment (kyranjamie) [inline src/app/features/settings-dropdown/settings-dropdown.tsx]: You can use the helper method `openInNewTab`

File: src/app/features/settings-dropdown/settings-dropdown.tsx [hunk 2]
```diff
@@ -101,6 +102,21 @@ export function SettingsDropdown() {
             >
               Change theme
             </MenuItem>
+            <MenuItem
+              data-testid={SettingsSelectors.ToggleTheme}
+              onClick={wrappedCloseCallback(() => {
+                void analytics.track('get_support_menu_item');
```
Comment (kyranjamie) [inline src/app/features/settings-dropdown/settings-dropdown.tsx]: Per convention should be a verb @markmhx `clicked_get_support_menu_item`?

Comment (markmhendrickson) [reply src/app/features/settings-dropdown/settings-dropdown.tsx]: image

File: src/app/features/settings-dropdown/settings-dropdown.tsx [hunk 3]
```diff
@@ -101,6 +102,21 @@ export function SettingsDropdown() {
...
+                window.open(
+                  'https://wallet.hiro.so/wallet-faq/where-can-i-find-support-for-the-stacks-wallet',
+                  '_blank'
+                );
+              })}
+            >
+              <Stack isInline>
+                <div>Get support</div>
```
Comment (fbwoolf) [inline src/app/features/settings-dropdown/settings-dropdown.tsx]: Use a `Box` component here?

**Review by fbwoolf — APPROVED** [review]
Can this be taken our of draft mode now?

**Review by kyranjamie — APPROVED** [review]

**Pull request merged**